### PR TITLE
GEOMESA-1374 Concat varargs and mkstring in converters

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <andrew.hulbert@ccri.com>

### CODE REVIEW

**Comment by jahhulbert-ccri** [comment]
i need to test the twitter thing really quick

File: geomesa-tools/conf/sfts/twitter/reference.conf [hunk 1]
```diff
@@ -175,20 +169,15 @@ geomesa  {
         { name = c3_lon, json-type = "string", path = "$.place.bounding_box.coordinates[0][3][0]", transform = "$0::double" }
         { name = c3_lat, json-type = "string", path = "$.place.bounding_box.coordinates[0][3][1]", transform = "$0::double" }
 
-        { name = c0, transform = "concat( $c0_lon, concat(' ', $c0_lat))" }
-        { name = c1, transform = "concat( $c1_lon, concat(' ', $c1_lat))" }
-        { name = c2, transform = "concat( $c2_lon, concat(' ', $c2_lat))" }
-        { name = c3, transform = "concat( $c3_lon, concat(' ', $c3_lat))" }
-        { name = c4, transform = "concat( $c0_lon, concat(' ', $c0_lat))" }
+        { name = co0, transform = "mkstring(' ', $c0_lon, $c0_lat)" }
```
Comment (jahhulbert-ccri) [inline geomesa-tools/conf/sfts/twitter/reference.conf]: oops

**Comment by jahhulbert-ccri** [comment]
ok...should be ready....

File: geomesa-tools/conf/sfts/twitter/reference.conf [hunk 2]
```diff
@@ -124,20 +124,14 @@ geomesa  {
...
-        { name = p5, transform = "concat( $p4, '))')"             }
-
-        { name = geom, transform = "polygon($p5)" }
+        { name = c0, transform = "mkstring(' ', $c0_lon, $c0_lat)" }
+        { name = c1, transform = "mkstring(' ', $c1_lon, $c1_lat)" }
+        { name = c2, transform = "mkstring(' ', $c2_lon, $c2_lat)" }
+        { name = c3, transform = "mkstring(' ', $c3_lon, $c3_lat)" }
+        { name = c4, transform = "mkstring(' ', $c0_lon, $c0_lat)" } // close it with the first point again
+
+        { name = polyWkt, transform = "concat( 'polygon((', mkstring(',', $c0, $c1, $c2, $c3, $c4), '))')"}
```
Comment (jnh5y) [inline geomesa-tools/conf/sfts/twitter/reference.conf]: Would it be better to have a bbox command?

Comment (jahhulbert-ccri) [inline geomesa-tools/conf/sfts/twitter/reference.conf]: ? its wkt

Comment (jahhulbert-ccri) [inline geomesa-tools/conf/sfts/twitter/reference.conf]: eventually i'd like to add a create polygon from points type thing...

**Pull request merged**